src/haddock/libs/libcns.py

```diff
@@ -277,7 +183,8 @@ def prepare_multiple_input(pdb_input_list, psf_input_list):
     for pdb in pdb_input_list:
         input_str += f"coor @@{pdb}{linesep}"
         input_str += (
-            f"eval ($input_pdb_filename_{ncount}=" f' "{pdb}"){linesep}'
+            f"eval ($input_pdb_filename_{ncount}="
+            f' "{pdb}"){linesep}'
```

Comment:
Yeah. there are many places where that space is there. I will clean all that in another PR. because there are also many other calls to that can go through the `write_eval_line` function. In another PR.